### PR TITLE
Create model and migration for Books

### PR DESCRIPTION
**Pull request by fedecvz**
## Summary:
Create model and migration for Books

## Trello card:
https://trello.com/c/0A7pZZ0M/21-crear-modelo-de-book

### CODE REVIEW

File: app/models/book.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class Book < ApplicationRecord
```
Comment (juanisierra) [inline app/models/book.rb:1]: Add rails validations to model, if not you will get a DB exception instead of more expressive rails validation errors

**Review by juanisierra — CHANGES_REQUESTED** [review]

File: db/migrate/20180828133820_create_books.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class CreateBooks < ActiveRecord::Migration[5.2]
+  def change
+    create_table :books do |t|
+      t.string :genre, :null => false
+      t.string :author, :null => false
+      t.string :image, :null => false
+      t.string :title, :null => false
+      t.string :publisher, :null => false
+      t.string :year, :null => false
```
Comment (juanisierra) [inline db/migrate/20180828133820_create_books.rb]: Why would the year be a string?

File: app/models/book.rb (hunk 1 above)
Comment (fedecvz) [reply app/models/book.rb:1]: The validations are in the book-test branch.

File: db/migrate/20180828133820_create_books.rb (hunk 2 above)
Comment (fedecvz) [reply db/migrate/20180828133820_create_books.rb]: You are rigth, it would be a int!! :)

**Pull request closed without merging**